README.md

```diff
@@ -1 +1,42 @@
+* [Learn about building and running the refine.bio project source code](https://github.com/AlexsLemonade/refinebio).
+
+`The Single-cell Pediatric Cancer Atlas (ScPCA)` focuses on single-cell profiling with the goal of creating a publicly available atlas of single-cell pediatric cancer data. Ten ScPCA awards were funded by Alex’s Lemonade Stand Foundation. The ALSF funded researchers submit their single-cell, single-nuclei, and bulk RNA sequencing data to the Data Lab for processing. The data from these patient tumors will be made openly available in one location through the ScPCA Portal, which is currently under development. 
+
+* [Management and analysis tools for ScPCA data.](https://github.com/AlexsLemonade/alsf-scpca)
+* [ScPCA tools](https://github.com/AlexsLemonade/scpcaTools) and [data files for testing them.](https://github.com/AlexsLemonade/scpcaData)
+* [User information about ScPCA processing.](https://github.com/AlexsLemonade/scpca-docs)
+
+`Open Pediatric Brain Tumor Analysis (OpenPBTA)` is a global open science initiative, which analyzes a vast collection of pediatric brain tumor data, comprising data from 943 tumors. This project operates on an open contribution model, crowdsourcing expertise from childhood brain cancer experts from across the world. 
+
+* [See the collaborative analysis in real-time.](https://github.com/AlexsLemonade/OpenPBTA-analysis)
+* [View the collaboratively written manuscript.](https://github.com/AlexsLemonade/OpenPBTA-manuscript)
+
+# Training Workshops
+
+We offer [`training workshops`](http://ccdatalab.org/training) to teach researchers the data science skills they need to examine their own data. Participants are introduced to the R programming language and to cutting-edge technologies used in single-cell and bulk RNA-sequencing data analysis. All of our training materials are openly licensed and freely available!
+
+* [View our current training modules.](https://github.com/AlexsLemonade/training-modules) 
+* [Sign up to be notified about future training workshops.](https://share.hsforms.com/1y55bYTSKSVKVOejXnM9lIg336z0) 
+* Questions? Interested in using our materials to hold your own workshop? Contact us at `training@ccdatalab.org`.
+
+# Get Involved
+
+Visit us at [`ccdatalab.org`](http://ccdatalab.org) and follow us on Twitter, [`@CancerDataLab.`](https://twitter.com/cancerdatalab)
+
+**We're hiring!** Are you interested in a unique opportunity to serve the childhood cancer research community? Visit our [`careers page`](https://www.ccdatalab.org/careers) to learn more about our current openings:
+
+* [Engineer](https://www.ccdatalab.org/engineer)
+* [Data Scientist](https://www.ccdatalab.org/data-scientist)
+
+Support our work by making a tax-deductible contribution to ALSF’s Childhood Cancer Data Lab. [`Donate here!`](http://ccdatalab.org/donate)
```

Comment:
Do we want to add a general way to contact us as the very last section? Probably would be `info@ccdatalab.org`.

It could also be a subsection of `# Get Involved`

Reply:
```suggestion
Support our work by making a tax-deductible contribution to ALSF’s Childhood Cancer Data Lab. [**Donate here!**](http://ccdatalab.org/donate)
```